src/layouts/Default.vue

```diff
@@ -129,13 +132,41 @@ export default {
         { name: "MySQL Database", route: "/docs/services/mysql" },
         { name: "Software Mirrors", route: "/docs/services/mirrors" },
         { name: "High Performance Computing", route: "/docs/services/hpc" }
-      ]
+      ],
+      darkMode: false
     };
   },
   computed: {
     user() {
       return store.user;
     }
+  },
+  watch: {
+    darkMode(newMode) {
+      if (
+        document.getElementById("app").classList.value.includes("dark") !==
+        newMode
+      )
+        document.getElementById("app").classList.toggle("dark");
```

Comment:
Querying elements can be placed into variables instead of repeating the statements; ie
```javascript
const appClassList = document.getElementById('app').classList;
```

Reply:
done! :)